onnx/defs/experiments/experiments_functions.cc

```diff
@@ -8,7 +8,7 @@ using namespace ONNX_NAMESPACE;
 
 static Common::Status BuildMVN(std::unique_ptr<FunctionProto>* func_proto) {
   if (nullptr == func_proto) {
-    return Status(
```

Comment:
remove the line of code "func.set_status(OperatorStatus::EXPERIMENTAL);" in this file to promote "MVN" as a production function.

Reply:
and set since version to "9" for this production function.